packages/core/cli/src/discovery.ts

```diff
@@ -1,29 +1,43 @@
 import * as readPkgUp from 'read-pkg-up';
 import * as resolveFrom from 'resolve-from';
-import { PluginSystem, PLUGIN_REGEX } from '@manta-style/core';
+import { PLUGIN_REGEX } from '@manta-style/core';
 
-export default class PluginDiscovery {
-  static async findPlugins(file: string) {
-    const { pkg } = await readPkgUp({ cwd: file, normalize: true });
-    const plugins = [
-      ...filterDependency(pkg.dependencies),
-      ...filterDependency(pkg.devDependencies),
-    ];
+const DEFAULT_PLUGINS = [
+  '@manta-style/server-restful',
+  '@manta-style/mock-example',
+  '@manta-style/mock-range',
+];
 
-    return new PluginSystem(
-      plugins.map((plugin) => {
-        return {
-          name: plugin,
-          module: defaultInterops(require(resolveFrom(file, plugin))),
-        };
-      }),
-    );
-  }
+export async function findPlugins() {
+  const file = process.cwd();
+  const { pkg } = await readPkgUp({ cwd: file, normalize: true });
+  const plugins = [
+    ...filterDependency(pkg.dependencies),
+    ...filterDependency(pkg.devDependencies),
+  ];
+  return [
+    ...defaultPlugins(DEFAULT_PLUGINS),
+    ...plugins.map((plugin) => {
+      return {
```

Comment:
just nitpicking here, feel like `DEFAULT_PLUGINS`, and `plugins` should use the similar function, for easier reading, like doing both `[...DEFAULT_PLUGINS.map(...), ...plugins.map(...)]` here, or both call a function, ie: `[...defaultPlugins(DEFAULT_PLUGINS), ...customPlugins(plugins, file)]`

Reply:
Fixed!